src/app/@ansyn/ansyn/modules/overlays/overlay-status/overlay-status.component.ts

```diff
@@ -189,8 +192,19 @@ export class OverlayStatusComponent implements OnInit, OnDestroy, IEntryComponen
 		if (this.isDragged) {
 			this.toggleDragged();
 		}
+
+		this.onClickOutSide$.unsubscribe();
 	}
 
+	ngAfterViewInit() {
```

Comment:
maybe we can change `Autosubscriptions` init function to `ngAfterViewInit`

Reply:
@tltk90 excellent idea

@007quardo please do not remove AutoSubscription. Even if we can't do what @tltk90 suggested, we get just raise a flag in ngAfterViewInit(), and then check the flag in the observable in AutoSubscription